### PR TITLE
Use golang:1.12-alpine3.9 for buildtool-alpine

### PR DESCRIPTION
**Pull request by UladzimirTrehubenka**

### CODE REVIEW

**Comment by Ri0n** [comment]
I remember last attempt to use newer golang version in buildtool caused some crashes in on-prem software.
@vasily-chertkov knows details.

**Comment by UladzimirTrehubenka** [comment]
For now NGP team (ATC team too - for PE and ATC-API projects) is using buildtool-alpine:latest based on golang:1.11.0-alpine3.8 w/o any issues.
Upgraded version from the PR has tag buildtool-alpine-3.9:1.12 instead buildtool-alpine:latest,
i.e. any project must set buildtool version explicitly (not latest).

**Review by dkamenda — APPROVED** [review]

**Review by dkamenda — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -15,8 +15,8 @@ clean:
 
 
 latest-alpine:
-	docker build -t $(PROJECT_APLINE):latest buildtool-alpine
+	docker build -t $(PROJECT_APLINE)-3.9:1.12 buildtool-alpine
```
Comment (Ri0n) [inline Makefile:18]: I believe it's better to keep the version in a variable

1) to not forgot to chang in multiple places
2) to allow override from comamnd line if it ever necessary.

Maybe even sed Dockerfile (or use Dockerfile.template) to replace version there

**Review by Ri0n — APPROVED** [review]

**Comment by UladzimirTrehubenka** [comment]
@dkamenda Could you please merge the PR and build the image?

**Comment by dkamenda** [comment]
> @dkamenda Could you please merge the PR and build the image?

Unfortunately I could not push the image: `requested access to the resource is denied`

**Pull request merged**